build-lualib.js

```diff
@@ -6,11 +6,8 @@ const tstl = require("./src");
 const { loadLuaLibFeatures } = require("./src/LuaLib");
 
 const configFileName = path.resolve(__dirname, "src/lualib/tsconfig.json");
-const { emitResult, diagnostics } = tstl.transpileProject(configFileName);
-emitResult.forEach(({ name, text }) => ts.sys.writeFile(name, text));
```

Comment:
This is no longer emitted? Doesn't this break lualib inlining?

Reply:
`transpileProject` now emits files itself (unless `writeFile` callback parameter is provided)